be/src/exec/pipeline/pipeline_driver_executor.cpp

```diff
@@ -84,8 +84,6 @@ void GlobalDriverExecutor::_worker_thread() {
                 continue;
             }
 
-            // query context has ready drivers to run, so extend its lifetime.
-            query_ctx->extend_lifetime();
```

Comment:
in QueryContextManager::remove function, extend_lifetime invocation also needs to be removed.

Reply:
What's the case?